.github/ISSUE_TEMPLATE/4_pre_release.yml

```diff
@@ -115,6 +115,7 @@ body:
     attributes:
       label: Releasing
       value: |
+        - [ ] Check hanging sidecars `simcore-service-director-v2 close-and-save-service --skip-container-removal --skip-state-saving --skip-outputs-pushing <uuid>`

```

Comment:
hmmm. that command does not save anything. it is a bit doubtful that this is always what you wanna do.. please first check if the sidecar should be saved properly

Reply:
yes of course ideally you want to run it without those additional options. I put it here just to have a helper in the doc with all the available options when there is some problem so you are able to solve it. But if you think is confusing/or can be misused I can remove the additional options.